### ISSUE TITLE
The "Onboarding" issues - educational bubbles pointing to wrong places

### ISSUE THREAD

**Issue by zxyanliu** [microsoft/BotFramework-Composer#2790]
## Describe the bug
Educational bubbles are pointing to the wrong places: 

* **Triggers**
![image](https://user-images.githubusercontent.com/32497439/80403577-a986f780-8874-11ea-9809-1e6e68e1b6fb.png)

* **Actions**
![image](https://user-images.githubusercontent.com/32497439/80403683-d804d280-8874-11ea-9243-d5dccfa800d5.png)

* **Add welcome message**

![image](https://user-images.githubusercontent.com/32497439/80403798-0f737f00-8875-11ea-80a2-613ecac6230d.png)

* **Add an intent trigger**
![image](https://user-images.githubusercontent.com/32497439/80403851-24e8a900-8875-11ea-8324-ece4bba02058.png)

## Version
Latest master branch. 

## Browser

- [x] Chrome

## OS

- [x] Windows

## To Reproduce

1. Enable the "Onboarding" functionality ([doc](https://docs.microsoft.com/en-us/composer/onboarding))
2. Start the "Onboarding tour"
3.  See errors.

## Expected behavior
Educational bubbles should point to the right places. 

## Additional context
Educational texts need polish.

**Comment by cwhitten** [comment]
@zxyanliu I'll see what we can do here. If we updated the Onboarding bubble content to be be more descriptive would this suffice for you?

**Comment by zxyanliu** [comment]
Hi @cwhitten 
* I filed this issue because we have [an article](https://docs.microsoft.com/en-us/composer/onboarding) on the onboarding experience in Composer and I should update the content that will make sense to our audience. 
* I realized the reason why the educational bubbles are pointing to the wrong places: it is because we added the "add" button in the tool bar and the flow to create a new dialog and a new trigger has been changed. 
* Yes, updating the onboarding bubble content is a great solution. Thanks a lot!
